tests/test_recovery_action.cpp

```diff
@@ -0,0 +1,24 @@
+#include "sm_tests.h"
+
+/** @brief Fills SWS_Platform_Health_Management] */
+TEST_F(smTests, testRecoveryAction)
+{
+    std::string retVal;
+    bool errorStatus;
+
+    errorStatus = mySM.recoveryAction.recoveryActionHandler(&retVal);
+    EXPECT_EQ(errorStatus, false);
```

Comment:
Don't do that here - this function shall be called from SM. Instead, use EXPECT_CALL (if you can do it without mock, I don't think so lol)